.gitlab/pkg_metrics.yml

```diff
@@ -221,7 +221,7 @@ check_pkg_size-a6:
     # "25000000" is needed as of now because of a large bump in size in system-probe in 7.35
     - |
         declare -Ar max_sizes=(
-            ["datadog-agent"]="25000000"
+            ["datadog-agent"]="50000000"
```

Comment:
I don't think we can just blindly increase these.

Reply:
@DataDog/agent-platform what is the procedure here?

Reply:
I already pinged them on slack
initial discussion here : https://dd.slack.com/archives/CNY5XCHAA/p1677161734490679